apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -247,14 +285,29 @@ export default class ExportHTMLModal extends Vue {
         StatusCountModule.countOf(this.filter, 'PassingTestsFailedControl') +
         StatusCountModule.countOf(this.filter, 'FailedTests')
     };
+    this.outputData.severity = {
+      low: SeverityCountModule.low(this.filter),
+      medium: SeverityCountModule.medium(this.filter),
+      high: SeverityCountModule.high(this.filter),
+      critical: SeverityCountModule.critical(this.filter)
+    };
+    this.outputData.compliance =
```

Comment:
<img width="228" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/9cdc7907-022e-468f-81a8-ec66adf3a4b2">

should probably truncate this to two decimal points

see https://github.com/mitre/heimdall2/pull/4655

Reply:
Should be good now